### PR TITLE
Call area.clear() from afterEach in tests

### PR DESCRIPTION
**Pull request by michielbdejong**
This also corrects the way Mocha is called (see https://mochajs.org/#arrow-functions)

### CODE REVIEW

File: test/chrome_test.js [hunk 1]
```diff
@@ -10,11 +10,11 @@ chai.config.includeStack = true;
 
 
 /** @test {StorageChange} */
-describe("StorageChange", () => {
+describe("StorageChange", function() {
 
   /** @test {StorageChange#constructor} */
-  describe("#constructor", () => {
-    it("should expose oldValue and newValue", () => {
+  describe("#constructor", function() {
```
Comment (n1k0) [inline test/chrome_test.js:16]: Any specific reason for dropping arrow functions here?

Edit: ok my bad missed the PR description.

File: test/chrome_test.js [hunk 2]
```diff
@@ -62,9 +64,7 @@ describe("StorageArea", () => {
       }).catch(done);
     });
 
-    it("set/remove a value in IDB", (done) => {
-      const area = new StorageArea("http://localhost:8080/v1/", "sync");
-
+    it("set/remove a value in IDB", function(done) {
       area.set({"something": 1}, function () {
         area.remove("something", function() {
           area.get("something", function(items) {
```
Comment (n1k0) [inline test/chrome_test.js:70]: Huh, callbacks. That's probably not part of this patch but have you considered using something like [promisify](https://www.npmjs.com/package/promisify) for these?

**Comment by n1k0** [comment]
LGTM

File: test/chrome_test.js (hunk 2 above)
Comment (michielbdejong) [inline test/chrome_test.js:70]: We could use promisify to write the tests for the callbacks-based storage.sync API, but I doubt it would make them more readable.

In the code, we trigger these callbacks from the promises returned by Kinto, so there we would benefit from using some sort of callbackify :) e.g. https://github.com/Kinto/kinto-chrome/blob/master/lib/chrome.js#L115-L117

**Comment by michielbdejong** [comment]
Thanks! Can you merge?
